VERSION-RANGE-SPEC.rst

```diff
@@ -637,6 +641,8 @@ These are a few known versioning schemes for some common Package URL
 - **alpine**: Alpine linux https://gitlab.alpinelinux.org/alpine/apk-tools/-/blob/master/src/version.c
   which is using Gentoo-like conventions.
 
+- **ecosystem-independent versioning schemes**: Several versioning schemes that are independent from any specific ecosystem are being developed. A very simple one is the generic *all* and *none* scheme which allows to specify that all (or none) of the versions are affected.
```

Comment:
That shouldn't be part of the list of "some common Package URL `types`".

Moreover, we are not talking about `affected` or `not affected`. `vers` is solely about "is it in the range or not". 
```suggestion

Moreover, there are several versioning schemes that are independent from any specific ecosystem.

- **all**: a generic versioning scheme where every version is inside the range, i.e. all versions match. This is equivalent to `true`.
- **none**: a generic versioning scheme where no version is inside the range, i.e. no versions match. This is equivalent to `false`.

Reply:
Also, we should sort this entries...